cadence/values.yaml

```diff
@@ -57,6 +57,65 @@ server:
   affinity: {}
 
   config:
+    # clusterMetadata contains the settings of the Cadence clusters
+    # participating in a Cadence cluster group.
+    clusterMetadata:
+      # enableGlobalDomains when set to true allows creating global domains
+      # which are replicated between all clusters and whose tasks are served by
+      # the active cluster in the replication.
+      #
+      # WARNING: This value **MUST NOT** be changed after launching the Cadence
+      # cluster group.
+      enableGlobalDomain: true
+
+      # failoverVersionIncrement is the increment of each cluster version when
+      # failover happens.
+      #
+      # WARNING: For each cluster, (initialFailoverVersion +
+      # i*failoverVersionIncrement) **MUST** result in a unique value.
+      failoverVersionIncrement: 10
+
+      # masterClusterName is the name of the only cluster which can register a
+      # domain, although each cluster can update domains and do domain failover.
+      #
+      # WARNING: This value **MUST NOT** be changed after launching the Cadence
```

Comment:
One more comment here:
masterClusterName can be changed. 
The only rule is that within a group, there is at most one `masterClusterName` at all time and they must have the same value for all clusters in the group. 

However, each cluster's currentClusterName cannot be changed(it's for determining which entry of `clusterInformation` is used. 

I hope it makes sense. It's indeed a complicated feature and I will add more documentation in https://cadenceworkflow.io/ later

Reply:
I attempted to fix it in https://github.com/banzaicloud/banzai-charts/compare/e18536f38ea7d0e1491f409fc4359e11c22178d8..b312d5e406780d95de7eee0c34dccfe1204ba751.

Yeah, this was a mistake on my end. I just haven't put 2 and 2 together from the definitions so the master is just for the privileged actions (domain registration ATM) and current cluster is for the active failover which has to be synced between the clusters so exactly 1 cluster is serving as active.